config/jobs/kubernetes-sigs/sig-windows/sig-windows-sac.yaml

```diff
@@ -46,13 +46,13 @@ periodics:
       - --aksengine-deploy-custom-k8s
       - --aksengine-agentpoolcount=2
       # Specific test args
-      - --test_args=--node-os-distro=windows --docker-config-file=$(DOCKER_CONFIG_FILE) --ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota|\[sig-network\].EndpointSlice  --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Slow\]|\[alpha\]|GMSA|Guestbook.application.should.create.and.stop.a.working.application
+      - --test_args=--node-os-distro=windows --docker-config-file=$(DOCKER_CONFIG_FILE) --ginkgo.focus=$(GINKGO_FOCUS_PARALLEL) --ginkgo.skip=$(GINKGO_SKIP_PARALLEL)
       - --ginkgo-parallel=4
       securityContext:
         privileged: true
   annotations:
-    testgrid-dashboards: sig-windows-sac, sig-windows-azure
-    testgrid-tab-name: aks-engine-azure-windows-2004-master
+    testgrid-dashboards: sig-windows-master-release
```

Comment:
I don't think we want the SAC test passes to show up in sig-windows-master-release, at least not the aks-engine ones.
Can we keep the sig-windows-sac dashboard?
Or maybe split it up into sig-windows-{sac-{versions}}?

Reply:
I was thinking the same with the hyper-v tests

Reply:
I like keeping the `sig-windows-sac`

Reply:
Sounds good, let me add it back